content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+- I’m lucky to teach in a computer lab -- I can coordinate with IT ahead of time to have R and RStudio installed on every computer (yes, time and labor intensive for someone). I have my most recent exercises and assignments posted online for them to download at the start and then run Wi-Fi free **(minimal Wi-Fi dependence)**. I also put my scripts on thumb drives **(Wi-Fi independent)**, for the worst case scenario – zero Wi-Fi. 
+- For the future, when students might bring in their own computers, I’m working on (and asking others to help) write bash scripts for different machines, which install R, Rstudio, and several base packages. I can distribute these bash scripts, and R scripts for the day’s exercises on thumb drives **(total Wi-Fi independence)** and have students go through dreaded installations with minimal pain. In any case, I do some intros and demos (fun) before installations begin (not fun).
+
+For **presentations**, I have a powerful mini projector with blue tooth connection (an Anker Capsule, many thanks to NIH research funding). So regardless of the awkward shape or layout of the room available to me, or however weak the projector is, I know I have my own flexible little powerhouse whose ins-and-outs I’m intimately familiar with.  
+
+There are two other workarounds (both Wi-Fi dependent) for students to watch **live coding** when visibility is poor. First is an institutional Zoom account that allows me to share my screen with several students. Another is the new `livecode` package from Colin Rundel that I’m excited to try. Check out the demo from Rstudioconf::2020 [here](https://resources.rstudio.com/rstudio-conf-2020/lightning-talk-colin-rundel). Essentially, your RStudio IDE is constantly creating a web page of your work, which your students can watch with the appropriate link.
+
+# **3. Finally, a make or break attitude for teaching beginners**
+
+We generally know, but regularly need to remember... When we first saw R or an RStudio IDE it was daunting and foreign. We may have since grown complacent and/or proud, having conquered many tasks through hours (or years) of debugging and google searches. Despite this, we can't give in to the temptation to trivialize, oversimplify, or show off skills in front of students. Our goal is to educate - not to impress and not to be lofty beacons of knowledge. In my experience, flashy shows and rapid demos often succeed at disheartening talented people. Knowledge and skills will feel unattainable to all but a very small percentage of type-A students. 
```

Comment:
I would rewrite this paragraph: "When we first saw R or the RStudio IDE it was daunting and foreign. Hours (or years) of debugging and Google searches may have made us forget this, but we must resist the temptation to trivialize or oversimplify in front of students. In my experience, flashy demos can often dishearten even talented people, making knowledge and skills feel unattainable to all but a few type-A students."